### ISSUE TITLE
`ClassVar` that can't be inferred should show an error

### ISSUE THREAD

**Issue by KotlinIsland** [KotlinIsland/basedmypy#587]
```py
class A:
    a: ClassVar = []  # expected error: cannot infer type, add explicit annotation
```
